controllers/glanceapi_controller.go

```diff
@@ -777,6 +771,47 @@ func (r *GlanceAPIReconciler) reconcileNormal(ctx context.Context, instance *gla
 	}
 	// create StatefulSet - end
 
+	// create ImageCache cronJobs
+
+	if len(instance.Spec.ImageCache.Size) > 0 {
+		// If image-cache has been enabled, create two additional cronJobs:
+		// - CacheCleanerJob: clean stalled images or in an invalid state
+		// - CachePrunerJob: clean the image-cache folder to stay under ImageCacheSize
+		//   limit
+		for _, item := range []glance.CronJobType{glance.CacheCleaner, glance.CachePruner} {
+			ctrlResult, err = r.ensureImageCacheJob(
+				ctx,
+				helper,
+				instance,
+				GetServiceLabels(instance),
+				serviceAnnotations,
+				item,
+			)
+			if err != nil {
+				instance.Status.Conditions.Set(condition.FalseCondition(
+					condition.CronJobReadyCondition,
+					condition.ErrorReason,
+					condition.SeverityWarning,
+					condition.CronJobReadyErrorMessage,
+					err.Error()))
+				return ctrlResult, err
+			}
+		}
+		// Cleanup any existing (but not used) ImageCache cronJob
+		if ctrlResult, err := r.deleteImageCacheJob(
```

Comment:
If I am not wrong, it will only cleanup CachePruner cronJob.

Reply:
It will clean both [1]

[1] https://github.com/openstack-k8s-operators/glance-operator/pull/435/files#diff-fe88ad2f79de701bd18c4a83352465de46564ebf02e97c1f6e6c1e0bd3f66554R1136

Reply:
I was under impression that ctrlResult is from line 781 loop, but its new one, so makes sense now.